llvm/test/tools/llvm-cov/showLineExecutionCounts.test

```diff
@@ -16,8 +17,10 @@
 //
 // Test html output.
 // RUN: llvm-cov show %S/Inputs/lineExecutionCounts.covmapping -format html -o %t.dir/html -instr-profile %t.profdata -path-equivalence=/tmp,%S/Inputs
+// RUN: llvm-cov show %S/Inputs/lineExecutionCounts.covmapping -format html -o %t.dir/html.binary -binary-counters=true -instr-profile %t.profdata -path-equivalence=/tmp,%S/Inputs
```

Comment:
Just `-binary-counters`. `=true` is a weird syntax that most command line parsing libraries don't allow.

Reply:
I've dropped `=true`. I like it, though.